### ISSUE TITLE
[COGNITION]: Accept Terms Prompt - SHOULD have a way to override the heading level

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10454]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [accept terms prompt](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/accepttermsprompt/) has a hard coded H1 in the component. This could lead to issues where we have multiple H1s on a page, and would benefit from having a way to pass in a `level` prop to adjust the heading.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Component only has one tab stop (identified by the yellow focus halo) that surrounds the entire component. Next focusable item should be the checkbox, then buttons.
- [ ] I can pass in a different heading level and verify the correct heading (H1, H2, H3...) appears in the rendered markup
- [ ] Visual styling remains consistent

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/accepttermsprompt/

## Possible Fixes (optional)
The alert box has a good example how we could fix this on [line 58](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/33cfe5e555c248b1920da58ddd7e47e5a02fc725/packages/formation-react/src/components/AlertBox/AlertBox.jsx#L58). We would want to add a default level, and a function to only allow values between 1 and 6.

```diff
! AcceptTermsPrompt.jsx#L100

+ const H = `h${this.props.level}`;
...
- <h1>{terms.title}</h1>
+ <H level={1}>{terms.title}</H>
```

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #217](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/217) via [**ZenHub**](https://www.zenhub.com/)
